src/pages/library/datasetBuilder/CohortEditor.test.ts

```diff
@@ -43,7 +51,14 @@ describe('CohortEditor', () => {
       conceptCount: 0,
       values: ['value'],
     });
-    render(createCriteriaViewComponent(_.noop)(criteria));
+    render(
+      h(CriteriaView, {
+        deleteCriteria: _.noop,
+        updateCriteria: _.noop,
+        criteria,
+        key: criteria.id,
+      })
```

Comment:
Is there a way to avoid the duplicate code here? Maybe add back the function for the test only?